pipelines/sbom-tool-main-build.yaml

```diff
@@ -232,23 +211,7 @@ extends:
             name: sbom-linux-build-pool
             os: linux
           steps:
-          - task: UseDotNet@2
-            displayName: 'Use .NET Core'
-            inputs:
-              useGlobalJson: true
-
-          - task: DotNetCoreCLI@2
-            displayName: 'Restore solution'
-            inputs:
-              command: restore
-              feedsToUse: config
-              nugetConfigPath: nuget.config
-              verbosityRestore: Normal
-
-          - task: DotNetCoreCLI@2
-            displayName: Build
-            inputs:
-              arguments: '-c $(BuildConfiguration)'
+          - template: pipelines/build-test-tool-template.yaml@self
```

Comment:
Odd that we were originally only running the tests on the Windows build. I like seeing them run on all platforms 😃

Reply:
Agreed!